tests/makefile.in

```diff
@@ -184,6 +184,9 @@ test11:
 	@echo "------------ TEST 11 (run unit tests)"
 	$(MAKE) ut
 
+coverage:
```

Comment:
should be called `utcoverage` or something like that because `coverage` is too general.